test/e2e/e2e_test.go

```diff
@@ -29,12 +32,13 @@ const (
 	exGwAnnotation       = "k8s.ovn.org/hybrid-overlay-external-gw"
 )
 
-func checkContinuousConnectivity(f *framework.Framework, nodeName, podName, host string, port, timeout int, podChan chan *v1.Pod, errChan chan error) {
+func checkContinuousConnectivity(f *framework.Framework, nodeName, podName, host string, port int, podChan chan *v1.Pod, errChan chan error) {
+	defer ginkgo.GinkgoRecover()
 	contName := fmt.Sprintf("%s-container", podName)
 
 	command := []string{
 		"bash", "-c",
-		"set -xe; for i in {1..10}; do nc -vz -w " + strconv.Itoa(timeout) + " " + host + " " + strconv.Itoa(port) + "; sleep 2; done",
+		"set -xe; for i in {1..20}; do nc -vz " + host + " " + strconv.Itoa(port) + "; sleep 2; done",
```

Comment:
I' ve hit the problem of using timeout with netcat
```
+ nc -vz -w 2 172.18.0.5 80
Connection to 172.18.0.5 80 port [tcp/http] succeeded!
+ sleep 2
+ for i in '{1..20}'
+ nc -vz -w 2 172.18.0.5 80
nc: connect to 172.18.0.5 port 80 (tcp) timed out: Operation in progress
```

Reply:
ok, seems legit, it's on the ovn-kube node test that 
anyway, I'm switching it to curl to have a more realistic scenario, nc -vz is a tcp probe only